source/operatingsystems/linux/controlpanels/cpanel_ftp_account.md

```diff
@@ -0,0 +1,30 @@
+# Creating An FTP Account In cPanel
+
+You might need to create FTP accounts in cPanel, so that you and/or your developers can upload and download content from your website.  
+
+In order to do this, start by logging into the cPanel account that you want to create the FTP account for. [You can find our guide on this here!](cpanel_connect)  
+
+Now, navigate to:
+```
+Files >> FTP Accounts
+```
+
+![FTP Accounts](files/ftp_accounts_button.JPG)
+
+You will be looking at this page:
+
+![FTP Accpunt Page](files/ftp_accounts_page.JPG)
+
+Firstly, choose and input a username and password for your account.
+
+The next box is for "Quota". This determines how much bandwidth the FTP account can use. By default, this can be left as unlimited.
+
+The last box is "Directory". Unless the account you're adding should only be allowed to access a specific subdirectory of the site, this can be left as blank. This will allow the FTP user to access all the files in the site's home directory.
```

Comment:
These steps should likely be swapped to match the order on the screenshot.

Reply:
Resolved in 2216c79